samza-azure/src/main/java/org/apache/samza/system/eventhub/consumer/EventHubSystemConsumer.java

```diff
@@ -105,6 +106,7 @@
 
   // Overall timeout for EventHubClient exponential backoff policy
   private static final Duration DEFAULT_EVENTHUB_RECEIVER_TIMEOUT = Duration.ofMinutes(10);
+  private static final long DEFAULT_INITIALIZE_EVENTHUBSMANAGERS_TIMEOUT_MILLIS = Duration.ofSeconds(15).toMillis();
   private static final long DEFAULT_SHUTDOWN_TIMEOUT_MILLIS = Duration.ofSeconds(15).toMillis();
```

Comment:
On second thought, this timeout is too low; the shutdown is async of all partitions. The `DEFAULT_EVENTHUB_RECEIVER_TIMEOUT` would be a more conservative value. @lhaiesp What do you think?

Reply:
done